aws/resource_aws_fms_admin_account.go

```diff
@@ -0,0 +1,125 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/fms"
+
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceAwsFmsAdminAccount() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsFmsAdminAccountPut,
+		Read:   resourceAwsFmsAdminAccountRead,
+		Delete: resourceAwsFmsAdminAccountDelete,
+
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(1 * time.Minute),
+		},
+
+		Schema: map[string]*schema.Schema{
+			"account_id": {
+				Type:         schema.TypeString,
+				Required:     true,
```

Comment:
Can we instead make this `Optional: true` and use `meta.(*AWSClient).accountid` if its not provided? e.g.

```go
if v, ok := d.GetOk("account_id"); ok && v != "" {
  accountID := v.(string)
} else {
  accountID := meta.(*AWSClient).accountid
}
```